packages/orbit-design-tokens/src/dictionary/formatters/json-tokens.ts

```diff
@@ -2,13 +2,20 @@ import { Property } from "style-dictionary";
 
 import { getValue } from "../utils/get";
 
+const getDeprecatedToken = deprecatedToken => {
```

Comment:
Maybe it should be typed to define what interface it expects?

Reply:
I have still some on going small issues with typeof of the style-dictionary, so I'll try to resolve them a bit later 🙂